deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -374,6 +377,8 @@ def get(self, request, name, stage=None):
                 "csrf_token": get_token(request),
                 "display_stopping_hosts": DISPLAY_STOPPING_HOSTS,
                 "project_name_is_default": project_name_is_default,
+                "console_project_exist": console_project_exist,
```

Comment:
can we use "console_project_env_url" to determine this exist flag? so we pass in one less variable.